Add PaymentRequiredError

Heidr opened the pull request.

Heidr committed:
Add PaymentRequiredError

mediafinger commented:
Hey Heidr,

thanks for the addition - I hope it's not only for Hacktoberfest ;-) Just out of curiosity: how will you use this 402 `payment required` error?

Another thing: your editor and/or git setup seems to miss the **trim trailing whitespaces** setting. Your editing of the README added a few trailing whitespaces. Could you fix this and squash this changes into one commit?

Thx :)

_(And when you are done, please tell Till to remind me to merge this, as I am on vacation right now #vanlife)._

Heidr commented:
Hey, thanks for your comment. 

We'll need this error to handle a specific Omikron response.

The whitespaces actually got auto-removed by my editor when I added the error to README, but later I reverted changes to this file. Now they're reapplied.

Thanks again and enjoy your vacation :)

mediafinger commented:
Thank you for your contribution. I've released version 1.5.1 with your addition.
